synapse/config/server.py

```diff
@@ -155,9 +155,10 @@ def default_config(self, server_name, **kwargs):
             # The port to listen for HTTPS requests on.
             port: %(bind_port)s
 
-            # Local interface to listen on.
-            # The empty string will cause synapse to listen on all interfaces.
-            bind_address: ''
+            # Local addresses to listen on.
+            # This will listen on all IPv4 addresses by default.
```

Comment:
Can we have the example in here of how to listen on ipv6 too, even if commented out?  presumably

```
# - '::'
```